src/rules/no_mixed_spaces_and_tabs.rs

```diff
@@ -89,6 +89,33 @@ impl LintRule for NoMixedSpacesAndTabs {
       }
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows mixed spaces and tabs for indentation.
+
+Most code conventions require either tabs or spaces be used for indentation. Therefore, if a line of code is indented with both tabs and spaces, it's most likely a mistake of a developer.
+
+### Invalid:
+
+```typescript
+function add(x: number, y: number) {
+          return x + y; // indented with a tab + two spaces
+}
```

Comment:
Thanks for the fix again, but unfortunately, it still seems like it consists of spaces only. Perhaps your editor config forces you to use spaces (i.e. soft tabs), instead of hard tabs. If my guess is correct, I'd recommend you see the config of your editor and set it to hard-tabs (temporarily). Or please feel free to apply my suggestion below by clicking "Commit suggestion" button 👍 

```suggestion
function add(x: number, y: number) {
	  return x + y; // indented with a tab + two spaces
}
```